ReviewMonitor.xcodeproj/project.pbxproj

```diff
@@ -283,9 +283,9 @@
 				TargetAttributes = {
 					693F5FA21F61F23F00FD8859 = {
 						CreatedOnToolsVersion = 8.3.2;
-						DevelopmentTeam = CEBY93VF85;
```

Comment:
Let's not make any changes to the project code signing settings. Could you revert this file change?

Reply:
Ok, Understand that ;). Will do

Reply:
Should be OK. (I think)